mlflow/langchain/api_request_parallel_processor.py

```diff
@@ -223,33 +218,21 @@ def call_api(
                         response = self.lc_model.invoke(
                             prepared_request_json, config={"callbacks": callback_handlers}
                         )
-                elif isinstance(self.request_json, list) and isinstance(
-                    self.lc_model, runnables_supports_batch_types()
-                ):
-                    response = self.lc_model.batch(
-                        prepared_request_json, config={"callbacks": callback_handlers}
-                    )
                 else:
                     response = self.lc_model.invoke(
```

Comment:
Should we raise Exception here if request_json is a list? The invoke function won't return error but result is not correct 🤔

Reply:
No. A typical case it the chat model single input is a list, see the return value of `_transform_request_json_for_chat_if_necessary`